### ISSUE TITLE
cannot import name 'TabularPrediction'

### ISSUE THREAD

**Issue by galievaz** [aws-samples/automl-pipeline-with-autogluon-sagemaker-lambda#2]
Hi,
in first, thank you for providing of library, it's realy useful. 
When is executing training job in Sagemaker (code-free-automl), there is import error:

Traceback (most recent call last):
  File "autogluon-tab-with-test.py", line 13, in <module>
    from autogluon import TabularPrediction as task
ImportError: cannot import name 'TabularPrediction'

I guess that's why upgrade of new release of autogluon.
How can I resolve this?
Thanks

**Comment by azdaly** [comment]
Yep! Having the same issue - [this](https://auto.gluon.ai/stable/tutorials/tabular_prediction/tabular-quickstart.html) shows that TabularPrediction has been deprecated in favor for TabularDataset, TabularPredictor. I'll circle back if I successfully update the code!

**Comment by edgarinvillegas** [comment]
Here my solution:

1) Untar `sourcedir.tar.gz`
2) Edit `autogluon-tab-with-test.py`
Replace 
`os.system('pip install autogluon')` 
by
`os.system('pip install autogluon==0.0.15 autogluon.tabular "mxnet<2.0.0"')`
3) Tar again to `sourcedir.tar.gz`
4) Upload `sourcedir.tar.gz` to s3 as indicated in the instructions.

The next train job will now work succesfully.
